addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2047,8 +2048,12 @@ options.registry.HeaderNavbar = options.Class.extend({
         // should be done another way in another place somehow...
         const noHamburgerWidget = this.findWidget('no_hamburger_opt');
         const noHamburgerHidden = noHamburgerWidget.$el.hasClass('d-none');
-        if (noHamburgerHidden && noHamburgerWidget.isActive()) {
+        // The noHamburgerWidget has 2 methods, including one which will only
+        // have its correct value once the page is reloaded. Therefore, we need
+        // to make sure the method call is only done once.
+        if (noHamburgerHidden && noHamburgerWidget.isActive() && !this._hamburgerAutoUpdated) {
             this.findWidget('default_hamburger_opt').enable();
+            this._hamburgerAutoUpdated = true;
```

Comment:
Your commit message seems great but I don't understand the issue anyway :grimacing: Maybe we'll talk about it at the farm...